File: python/search/linear_search.py
import random as r

def linearSearch(mArray, key):
    for _ in mArray:
        if _ == key:
            return True
    return False

def main():

    a = []

    for i in range(100):
        a.append(r.randint(-100,100))

    a.sort()

    n = int(input("Enter a number: "))

    print(linearSearch(a,n))

if __name__ == "__main__":
    main()

File: javascript/search/linear_search.js

function linearSearch(arr, key){
	for(var i = 0; i < arr.length; i++){
		if(arr[i] === key){
			return true;
		}
	}
	return false;
}

var test = [0, 2, 12, 13, 9, 22];

console.log(linearSearch(test, 54));